core/views.py

```diff
@@ -56,20 +56,37 @@ def cadastrar_cliente(request):
             email = form_cliente.data['email']
             senha = form_cliente.data['senha']
 
+            dados_cep = None
+            try:
+                cep = requests.get(f"https://viacep.com.br/ws/{form_cliente.data['cep'].replace('-','')}/json/")
+                
+                dados_cep = cep.json()
+                if cep.status_code != 200 or dados_cep.get('erro', False):
```

Comment:
por que o if se o except faz a mesma coisa?

o `try` é exatamente para isso.... já tentou `catch`?  talvez `except` nao seja necessario

Reply:
Verdade, está ambíguo o `cep.status_code != 200`, mas o `dados_cep.get('erro')` da condição é necessário, pois o tipo de retorno é `{'erro': True}` e o bloco try não consegue detectar. Entretanto para não ficar duplicada as msgs vou forçar a entrada no exception com `raise Exception()` caso o `{'erro': True}`